pickATrip_django_apps/tests/test_pickATrip_django_apps_views.py

```diff
@@ -0,0 +1,18 @@
+from django.urls import reverse
+from django.test import Client
+from pytest_django.asserts import assertTemplateUsed
+
+
+class TestViews:
+    def setup(self):
+        self.client = Client()
+
+    def test_homepage_GET(self):
```

Comment:
Please refer this helper of django-pytest
https://pytest-django.readthedocs.io/en/latest/helpers.html#client-django-test-client
no need for setup :)
test proto -> test_homepage_GET(client) to use the helper